client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -116,6 +119,8 @@ object FlightsWithSplitsTable {
                     hasTransfer = props.hasTransfer,
                     indirectRedListPax = redListPaxInfo,
                     directRedListFlight = directRedListFlight,
+                    coachTransfer = props.coachTransfer,
+                    hasRedListOrigin = isRedListOrigin
```

Comment:
The `directRedListFlight` parameter actually already has a property `isRedListOrigin` so we don't need to add `hasRedListOrigin`

Reply:
yeah , Let me make changes and see .